Asterisk.2013/Asterisk.2013.sln

```diff
@@ -48,8 +52,23 @@ Global
 		{186124F7-82FA-4562-8B87-33D53A84E8BA}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{186124F7-82FA-4562-8B87-33D53A84E8BA}.Release|Any CPU.Build.0 = Release|Any CPU
 		{186124F7-82FA-4562-8B87-33D53A84E8BA}.Travis|Any CPU.ActiveCfg = Travis|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Release|Any CPU.Build.0 = Release|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Travis|Any CPU.ActiveCfg = Debug|Any CPU
+		{47640C6B-7F9E-43CD-8FF2-663CF2F8B033}.Travis|Any CPU.Build.0 = Debug|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Release|Any CPU.Build.0 = Release|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Travis|Any CPU.ActiveCfg = Debug|Any CPU
+		{7E816CCA-3766-4D08-8275-148E9D05FBC4}.Travis|Any CPU.Build.0 = Debug|Any CPU

```

Comment:
Might want to add a `Travis` configuration, so `Travis` isn't using `Debug`.
See: https://github.com/AsterNET/AsterNET/pull/147#pullrequestreview-131532982

Reply:
@mkchx, configuring "Travis" is something @skrusty has to do.
You just need to add a build configuration named `Travis` for the `Travis` build profile.
He had to do the same in https://github.com/AsterNET/AsterNET/pull/147#pullrequestreview-131532982, so see what he did.

Reply:
@mkchx, configuring "Travis" is something @skrusty has to do.
You just need to add a build configuration named `Travis` for the `Travis` build profile. (Or whatever these things are called.)
@lionsoft had to do the same in https://github.com/AsterNET/AsterNET/pull/147#pullrequestreview-131532982, so see what he did.